spec/black_box/black_box_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe BlackBox, type: :black_box do
   describe "#article_hotness_score" do
-    let!(:article) { build_stubbed(:article, published_at: Time.current) }
+    # let!(:article) { build_stubbed(:article, published_at: Time.current) }
```

Comment:
Should we just remove this instead of commenting it out?

Reply:
Forgot to remove it! Will do